tests/integration/workflows/test_arxiv_workflow.py

```diff
@@ -186,53 +293,98 @@ def test_harvesting_arxiv_workflow_manual_accepted(
-            record=record,
-        )
+    workflow_uuid, eng, obj = get_halted_workflow(
+        app=workflow_app,
+        extra_config={'PRODUCTION_MODE': False},
+        record=record,
+    )
 
-        do_accept_core(
-            app=workflow_app,
-            workflow_id=obj.id,
-        )
+    do_accept_core(
+        app=workflow_app,
+        workflow_id=obj.id,
+    )
 
-        eng = WorkflowEngine.from_uuid(workflow_uuid)
-        obj = eng.processed_objects[0]
-        assert obj.status == ObjectStatus.WAITING
+    eng = WorkflowEngine.from_uuid(workflow_uuid)
+    obj = eng.processed_objects[0]
+    assert obj.status == ObjectStatus.WAITING
 
-        response = do_robotupload_callback(
-            app=workflow_app,
-            workflow_id=obj.id,
-            recids=[12345],
-        )
-        assert response.status_code == 200
+    response = do_robotupload_callback(
+        app=workflow_app,
+        workflow_id=obj.id,
+        recids=[12345],
+    )
+    assert response.status_code == 200
 
-        obj = workflow_object_class.get(obj.id)
-        assert obj.status == ObjectStatus.WAITING
+    obj = workflow_object_class.get(obj.id)
+    assert obj.status == ObjectStatus.WAITING
 
-        response = do_webcoll_callback(app=workflow_app, recids=[12345])
-        assert response.status_code == 200
+    response = do_webcoll_callback(app=workflow_app, recids=[12345])
+    assert response.status_code == 200
 
-        eng = WorkflowEngine.from_uuid(workflow_uuid)
-        obj = eng.processed_objects[0]
-        # It was accepted
-        assert obj.status == ObjectStatus.COMPLETED
+    eng = WorkflowEngine.from_uuid(workflow_uuid)
+    obj = eng.processed_objects[0]
+    # It was accepted
+    assert obj.status == ObjectStatus.COMPLETED
+
+
+@mock.patch(
+    'inspirehep.modules.workflows.tasks.arxiv.download_file_to_workflow',
+    side_effect=fake_download_file,
+)
+@mock.patch(
+    'inspirehep.modules.workflows.tasks.beard.json_api_request',
+    side_effect=fake_beard_api_request,
+)
+@mock.patch(
+    'inspirehep.modules.workflows.tasks.magpie.json_api_request',
+    side_effect=fake_magpie_api_request,
+)
+def test_merge_with_already_existing_article_in_the_db(
+    mocked_download_arxiv,
+    mocked_api_request_beard,
+    mocked_api_request_magpie,
+    workflow_app,
+    mocked_external_services,
+):
+    head = record_insert_or_replace(read_file('fixtures', 'merger_head.json'))
+    es.indices.refresh('records-hep')
+
+    _insert_wf_record_source(
+        record_uuid=head.id,
+        source='arXiv',
+        json=read_file('fixtures', 'merger_root.json'),
+    )
+    es.indices.refresh('records-hep')
+    update = read_file('fixtures', 'merger_update.json')
+
+    obj_id = run_workflow(
+        app=workflow_app,
+        extra_config={
+            'ARXIV_CATEGORIES_ALREADY_HARVESTED_ON_LEGACY': [],
+            'PRODUCTION_MODE': False,
+        },
+        record=update,
+    )
+
+    do_accept_core(
```

Comment:
Use a new one like `resolve_conflicts` (very similar, but not passing any params to the endpoint).